src/main/java/org/apache/commons/io/input/ReaderInputStream.java

```diff
@@ -213,6 +230,16 @@ public void close() throws IOException {
         reader.close();
     }
 
+    /**
+     * The implementation by {@link ReaderInputStream} always returns 0.
+     */
+    @Override
+    public int available() throws IOException {
+        // Could implement this using Reader.ready(), but would be rather cumbersome
+        // and probably not very efficient
+        return 0;
+    }
+
```

Comment:
This PR is only supposed to be about docs as stated in the PR title. Furthermore this change is not matched with a unit test.

Reply:
This class directly extends `InputStream` and the [`InputStream.available()` documentation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/InputStream.html#available()) says:
> The available method of InputStream always returns 0.

I could have used `return super.available();` instead, but then it would be less obvious to the reader why the documentation of this method here says that it always returns 0 but calls `super.available()`. So I wanted to avoid requiring them to look up the super implementation to verify that this method behaves correctly.

Reply:
Please drop this method, it duplicates the superclass' implementation.

Reply:
ping.

Reply:
Sorry; have removed this method again now in the latest commit.